native/core/src/parquet/data_type.rs

```diff
@@ -48,6 +52,7 @@ make_type!(FLBADecimal32Type);
 make_type!(FLBADecimal64Type);
 make_type!(FLBAType);
 make_type!(Int32DateType);
+make_type!(Int32TimestampMicrosType);
```

Comment:
should we have `Int32TimestampMillisType` as well?

Reply:
We don't have a usage for mills for now I think